website/content/commands/partition.mdx

```diff
@@ -66,6 +73,16 @@ You can issue the following subcommands with the `consul partition` command.
 ### `create`
 
 The `create` subcommand sends a request to the server to create a new admin partition.
+This subcommand has the following characteristics:
+
+| Characteristic | Value |
+| :------------- | :---- |
+| [Required ACLs](/docs/security/acl/acl-system) | `operator:write` |
+| Corresponding HTTP API Endpoint | [\[PUT\] /partition](/api-docs/admin-partitions#create-a-partition) |
+| [Blocking Queries](/api-docs/features/blocking) | not supported from commands, but may be from the corresponding HTTP API endpoint |
+| [Agent Caching](/api-docs/features/caching) | not supported from commands, but may be from the corresponding HTTP API endpoint |
+
+#### Usage
```

Comment:
I added the "Usage" subheading because I thought the content looked strange otherwise. It just jumps from the characteristics table to a shell command box. See image below showing what it looks like with and without that heading. Let me know how you want me to proceed.

![image](https://user-images.githubusercontent.com/85913323/159790559-892e5e76-33ff-4aed-bce7-a73a0a930bed.png)

Reply:
@trujillo-adam as we discussed, I removed both the usage subheading **and** the usage command string. [Preview here](https://consul-602w17yc3-hashicorp.vercel.app/commands/partition).

It now looks like:
![image](https://user-images.githubusercontent.com/85913323/162082565-c7a5fc04-1453-47b0-aa3a-2c3ae9823c5f.png)